.github/workflows/deploy-dev.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - uses: ./.github/workflows/build
         with:
           secrets: ${{ toJSON(secrets) }}
-          prod: ${{ github.ref == 'refs/heads/main' }}
+          prod: ${{ github.base_ref == 'main' }} # PRs to main are Release Candidates and must be tested with prod settings
```

Comment:
Will this still work for both staging and dev deployments? Is base_ref dev when merging something from dev to main or only when merging into dev?